scripts/localeNames.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   'am-ET': 'Amharic',
+  ar: 'Arabic',
   'ar-EG': 'Arabic (Egypt)',
   'ar-SA': 'Arabic (Saudi Arabia)',
   'ar-SD': 'Arabic (Sudan)',
```

Comment:
Would this work https://github.com/mui/material-ui/issues/31847#issuecomment-2536857376?

```suggestion
  ar: 'Standard Arabic',
```